.github/workflows/complexity-check.yml

```
@@ -25,19 +25,21 @@ jobs:
         run: |
           echo "🔍 Analyzing test infrastructure complexity..."
           
-          # Count total lines in streamlined test framework
-          TOTAL_TEST_LINES=$(find tests -name "*.js" -exec wc -l {} + 2>/dev/null | tail -1 | awk '{print $1}' || echo 0)
+          # Count total lines in streamlined test framework (excluding E2E tests)
+          TOTAL_TEST_LINES=$(find tests -name "*.js" -not -path "tests/e2e/*" -exec wc -l {} + 2>/dev/null | tail -1 | awk '{print $1}' || echo 0)
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)